### PR TITLE
Added manifest

### PR DESCRIPTION
**Pull request by ideveCore**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.gitlab.idevecore.Pomodoro.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  ],
+  "modules": [
+    {
+      "name": "pomodoro",
+      "builddir": true,
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://gitlab.com/idevecore/pomodoro.git"
```
Comment (hfiguiere) [inline com.gitlab.idevecore.Pomodoro.json:34]: you need a tag or a commit here.

File: com.gitlab.idevecore.Pomodoro.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+  "app-id": "com.gitlab.idevecore.Pomodoro",
```
Comment (hfiguiere) [inline com.gitlab.idevecore.Pomodoro.json:2]: it should be `io.gitlab`.

see https://docs.flatpak.org/en/latest/conventions.html#application-ids

File: io.gitlab.idevecore.Pomodoro.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  ],
+  "modules": [
+    {
+      "name": "pomodoro",
+      "builddir": true,
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://gitlab.com/idevecore/pomodoro.git"
```
Comment (hfiguiere) [inline io.gitlab.idevecore.Pomodoro.json:34]: you need a tag or a commit here.

**Comment by hfiguiere** [comment]
bot, build io.gitlab.idevecore.Pomodoro

**Comment by hfiguiere** [comment]
```
builddir/files/share/appdata/io.gitlab.idevecore.Pomodoro.appdata.xml: FAILED:
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
• style-invalid         : Not enough <screenshot> tags, minimum is 1
• tag-missing           : <release> required
• tag-missing           : <name> is not present
• tag-missing           : <summary> is not present
• tag-missing           : <url> is not present
Validation of files failed
```

**Comment by ideveCore** [comment]
I think it finally works now, I'm new to publishing apps for flathub.

**Comment by hfiguiere** [comment]
bot, build io.gitlab.idevecore.Pomodoro

**Comment by ideveCore** [comment]
How long will it take for the app to appear on the Flathub website?

**Comment by hfiguiere** [comment]
it needs to be approved and merged. So it will take SOME TIME. Just be patient.

**Comment by barthalion** [comment]
bot, build io.gitlab.idevecore.Pomodoro

**Comment by barthalion** [comment]
This now fails because of the `.desktop` suffix in the ID: https://gitlab.com/idevecore/pomodoro/-/blob/main/data/io.gitlab.idevecore.Pomodoro.appdata.xml.in#L3. That regression has been already reported at https://github.com/ximion/appstream/issues/500

**Comment by ideveCore** [comment]
ready removed

**Comment by hfiguiere** [comment]
bot, build io.gitlab.idevecore.Pomodoro

**Pull request closed without merging**